frontend/src/components/dashboard/components/add-course-modal/add-course-modal.tsx

```diff
@@ -14,17 +14,19 @@ type Props = {
 };
 
 const AddCourseModal: FC<Props> = ({ isModalOpen, onModalToggle }) => {
-  const { control, errors, handleSubmit } = useAppForm<CourseCreateRequestDto>({
-    defaultValues: DEFAULT_CREATE_COURSE_PAYLOAD,
-    validationSchema: courseCreateValidationSchema,
-  });
+  const { control, errors, handleSubmit, reset } =
+    useAppForm<CourseCreateRequestDto>({
+      defaultValues: DEFAULT_CREATE_COURSE_PAYLOAD,
+      validationSchema: courseCreateValidationSchema,
+    });
 
   const dispatch = useAppDispatch();
 
   const onSubmit = (payload: CourseCreateRequestDto): void => {
     dispatch(dashboardActions.addCourse(payload))
       .unwrap()
-      .then(() => onModalToggle());
+      .then(onModalToggle)
+      .then(() => reset());
```

Comment:
Why don't to do it inside one handler?
```suggestion
      .then(() => {
        onModalToggle();
        reset();
      });
```

Reply:
`onModalToggle` is moved upper in #465, so there'll be one `.then` left :) But I'm thinking to fix those 2 issues inside of 1 PR, to avoid further conflicts